### ISSUE TITLE
indent - option to ignore ternary expressions

### ISSUE THREAD

**Issue by mjomble** [eslint/eslint#8725]
The `indent` rule supports ignoring many expression types using an `off` option, such as MemberExpression, FunctionDeclaration, CallExpression, etc.

It would be helpful if it could be similarly configured to ignore ternary expressions.
For example, I prefer a style for multi-line ternaries that is currently considered invalid by the `indent` rule:

```javascript
const value = (
  condition ?
  valueIfTrue :
  valueIfFalse
)
```

I tried the `flatTernaryExpressions` setting, but the indentation was still reported as incorrect.

Speaking of which, the `flatTernaryExpressions` setting seems quite out of place compared to the format of the other settings. Perhaps it could be replaced with one named `ConditionalExpression` (to match the name of the AST node) with options `off`, `flat`, etc?

Info on my environment:

* **ESLint Version:** 4.0.0
* **Node Version:** 6.9.1
* **npm Version:** yarn 0.24.6
* **Parser:** babel-eslint

**Comment by Pyrolistical** [comment]
Is this a dupe of https://github.com/eslint/eslint/issues/8712 ?

**Comment by not-an-aardvark** [comment]
> Is this a dupe of #8712 ?

Not quite. However, I think it will be redundant after https://github.com/eslint/eslint/issues/8594 is implemented.

**Comment by mjomble** [comment]
It does look like #8594 would take care of it. Shall I close this issue?

**Comment by not-an-aardvark** [comment]
Yes, I think I'll close it, since it would be better if we didn't add redundant options.
